functions/src/api/v1/invite/inviteParticipants.ts

```diff
@@ -0,0 +1,144 @@
+import { getAppEnv, getOVHEnv, getSendGridEnv } from '../../../firebase/env'
+import { NotificationContent } from '../../../types/Notification'
+import { sendNotifications } from '../../../notifications/sendNotifications'
+import { UID } from '../../../types/uid'
+
+/**
+ * @swagger
+ * /v1/rooms/{roomId}/sendInvites:
+ *   post:
+ *     description: Send an invitation to one or many participants, via email and/or sms. It can combine email and sms destinations, as well as specific lang for each participants. Email & phone can be used at the same time, email will be sent first. <br/> Default lang is "en". <br/> Available languages are en, fr, de, es, gr (el), hu, it, ro. <br/> Country are defaulted to "fr" if not supplied, it improve the phone umber parsing success rate, though it is already good by default (it use libphonenumber-js). <br/> Email and phone numbers are not saved in InstantVisio databases, not logged. Phone number are stored for a maximum of 24 hours on OVH Telecom (CRON run to clear them up every 12h), while emails logs are kept on SendGrid (email) without being able to erase those logs.
+ *     tags:
+ *       - rooms
+ *     consumes:
+ *     - multipart/form-data
+ *     produces:
+ *     - application/json
+ *     parameters:
+ *       - name: hostname
+ *         description: The name or organisation which sent the invite(s)
+ *         in: x-www-form-urlencoded
+ *         required: true
+ *         type: string
+ *       - name: destinations
+ *         description: An array of destinations
+ *         in: x-www-form-urlencoded
+ *         required: true
+ *         examples:
+ *            mixedTypeAndLang:
+ *                summary: Mixed email, sms and languages
+ *                value: [{email: "user@example.com", lang: "en"}, {phone: "+33600000000", lang:"fr"}, {phone: "+33600000000", lang:"fr", country:"en"}]
+ *            emailInvite:
+ *                summary: One email invite with French lang
+ *                value: [{email: "user@example.com", lang: "fr"}]
+ *         schema:
+ *            type: array
+ *            items:
+ *                type: object
+ *                properties:
+ *                   email:
+ *                       type: string
+ *                   phone:
+ *                       type: string
+ *                   lang:
+ *                       type: string
+ *                       default: en
+ *                       enum:
+ *                           - en
+ *                           - fr
+ *                           - de
+ *                           - es
+ *                           - gr
+ *                           - hu
+ *                           - it
+ *                           - ro
+ *
+ *     responses:
+ *       200:
+ *         description: One or many invites were sent successfully. It will return a 2xx if some invitation where not able to be sent (malformed email or phone number for eg).
```

Comment:
typo `It will return a 4xx`

Reply:
no, this was legit, if the user want to send 5 emails and only 1 is sent, it will return a 2xx